deploy/kustomize/olm-catalog/bases/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -109,23 +111,27 @@ spec:
     $ ipmctl create -goal PersistentMemoryType=AppDirect
     ```
     **Label the cluster nodes that provide persistent memory device(s)**
+
+    You can skip this step if your cluster has [Node Feature Discovery((NFD)](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html) installed.
     ```
-    $ kubectl label node <your node> storage=pmem
+    $ kubectl label node <your node> feature.node.kubernetes.io/memory-nv.dax="true"
     ```
-    
+
```

Comment:
We discussed that the setup instructions on operator hub should not repeat content from our main documentation site and instead guide users toward that.

Do you intend to implement that in this PR or separately?

Reply:
> Do you intend to implement that in this PR or separately?

I will prepare a separate PR.

Reply:
I'm not sure why we need to merge this PR, but okay.

I opened an issue so that we don't forget: https://github.com/intel/pmem-csi/issues/891